website/source/docs/job-specification/job.html.md

```diff
@@ -138,7 +148,45 @@ job "docs" {
       }
 
       resources {
-        cpu = 10
+        cpu = 20
+      }
+    }
+  }
+}
+```
+
+### Secrets Job
+
+This example shows a job which retrieves secrets from Vault and writes those
+secrets to a file on disk, which the application then consumes. Nomad handles
+all interactions with Vault.
+
+```hcl
+job "docs" {
+  datacenters = ["default"]
+
+  vault_token = "a3594cbc-dee6-40cb-a9e9-59dd5abf8985"
+
+  group "example" {
+    task "uptime" {
+      driver = "exec"
+
+      config {
+        command = "cat"
+        args    = ["local/secrets.txt"]
+      }
+
+      template {
+        data        = "{{ secret \"secret/data\" }}"
+        destination = "local/secrets.txt"
+      }
+
+      vault {
+        policies = ["secret-readonly"]
+      }
+
+      resources {
```

Comment:
I don't want to remove it because I think these should be copy-pastable jobs, and `resources` is required iirc. I still need to go through all the jobs and make sure they run as-is.